fiftyone/core/dataset.py

```
@@ -1105,6 +1108,193 @@ def deleted(self):
+
+        Raises:
+            ValueError: if the label schema or schemas do not exist
+        """
+        if fields is None:
+            fields = list(self.label_schemas)
+
+        for field in fields:
+            if field not in self._doc.label_schemas:
+                raise ValueError(
+                    f"field '{field}' is not in the dataset's label schema"
+                )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)